guides/FFI.md

```diff
@@ -85,14 +78,12 @@ The general rule regarding types is that you can enforce as little or as much ty
 In PureScript, JavaScript code is wrapped using a _foreign module_. A foreign module is just a CommonJS module which is associated with a PureScript module. Foreign modules are required to adhere to certain conventions:
 
 - The name of the foreign module must be the same as its companion PureScript module, with its extension changed to `.js`. This associates the foreign module with the PureScript module.
-- All exports must be of the form `exports.name = value;`, specified at the top level.
+- All exports must be of the form `export const name = value;`, specified at the top level.
```

Comment:
This is only true for values, not necessarily functions (as the example below breaks this rule).